scraper/items.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
documentation here is via the 'scrapy startproject' boilerplate, the class below is my own